public/assets/css/home.css

```diff
@@ -0,0 +1,85 @@
+@import url('https://fonts.googleapis.com/css2?family=Londrina+Solid&family=Roboto&family=Skranji&display=swap');
```

Comment:
This @import is useless (already exists in style.css).
Also as you know, this is a temporary solution as our fonts will be locally hosted in a upcoming commit (from me).

Reply:
Thanks for you feed back, correction done.